events/elections/2021/README.md

```diff
@@ -45,46 +45,79 @@ Eligibility for voting in 2021 is defined as:
 then please [request an exception][voter exemption form] (and please apply for
 Org Membership as well).
 
-Corporate affiliation is applied after the election. If an organization finds
-itself with too many representatives it is up to those individuals to come
-to a consensus on who should serve on the committee.
+Only contributions to projects and artifacts that fall under Steering
+Committee's governance will be considered for voter exemption.
+
+Examples of contributions that would be considered:
+* Slack admins who are not active in GitHub
+* Code of Conduct Committee members whose actions are private by default
+
+Examples of contributions that would NOT be considered:
+* Contributions to ecosystem projects and products
+* Organizing meetups or podcasts
 
 ### Schedule
 
+<!-- While finalizing the dates in the schedule, ensure that:
+- The Steering Committee and candidate Q+A occurs at a public SC meeting
+  (usually a Monday).
+- Dealine to submit voter exception forms and request a
+  replacement ballot is ~3 days before voting closes.
+- Private announcement of results to SC members is at least ~2 days
+  before private announcement to all candidates.
+- The interval between private announcement to all candidates and the
+  public announcement is a weekend.
+-->
+
 | Date         | Event                    |
 | ------------ | ------------------------ |
-| August XX    | Announcement of Election and publication of Voters.md |
+| July 1       | Steering Committee selects Election Committee |
+| August XX    | Announcement of Election and publication of voters.md |
 | August XX    | Steering Committee Meeting with Q+A with the candidates and community |
-| September XX | All candidate bios and voting exception forms due by 0000 UTC (5pm PST) |
+| September XX | All candidate bios due by 0000 UTC (5pm PST) |
 | ~1 week      | Election prep week (voters.md validation and CIVS setup and testing)
 | September XX | Election Begins via email ballots |
-| October XX   | Deadline to request a replacement ballot |
+| October XX   | Deadline to submit voter exception forms and request a replacement ballot |
 | October XX   | Election Closes by 0000 UTC (5pm PST) |
-| October XX   | Announcement of Results at Public Steering Committee meeting |
+| October XX   | Private announcement of Results to SC members not up for election |
+| October XX   | Private announcement of Results to all candidates |
+| October XX   | Public announcement of Results at Public Steering Committee Meeting |
+| October XX   | Election Retro |
 
 ## Candidacy Process
 
 **Nomination**
 
-If you want to stand for election, send an email to kubernetes-dev@googlegroups.com
-with the subject line "Steering Committee Nomination: Your Name (@yourgithub)".
+If you want to stand for the election, create an issue in this GitHub repo
+(kubernetes/community) with the title `Steering Committee Nomination: Your Name (@yourgithub)`.
```

Comment:
We talked about moving to GitHub PRs instead of issues, but I wasn't sure what the PR would change. Updating the list of nominees could lead to rebases again :/

Reply:
was the flow something like:
1. nominator opens issue `#NNN`, people seconding the nomination comment or +1 the issue
2. to accept, candidate comments on the issue and opens PR with bio file marked as `Fixes #NNN`)
3. election committee adds `/lgtm` + `/approve` to the bio PR and takes responsibility for adding the candidate to the index, avoiding rebase conflicts

Reply:
The PRs would be the PR for the candidate's Bio, of course.  That way asking them to write their bios isn't a seperate step.  We did this for Knative and it worked.

The drawback of that is that it makes it awkward for community members to nominate other people as candidates, but they can do that just by submitting a blank bio.

Reply:
if the nominator opens an issue, and the candidate opens a bio PR resolving the issue, that seems like a reasonable flow

Reply:
Yah.  I think the candidate themselves should be able to just open the PR without opening the issue though ... no?